samples/Calculator/csharp/windows/calculator/calculator.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- This project was created with react-native-windows 0.73.2 -->
+<!-- This project was created with react-native-windows 0.74.0 -->
 <Project ToolsVersion="Current" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <Import Project="$(SolutionDir)\ExperimentalFeatures.props" Condition="Exists('$(SolutionDir)\ExperimentalFeatures.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
-    <ProjectGuid>{7410B9FC-308E-4496-B556-FC56F1910AA7}</ProjectGuid>
+    <ProjectGuid>{B2ABB10D-1A83-4DF1-8D0C-7167F3F88A5F}</ProjectGuid>
```

Comment:
You should be able to do a find replace to put back the old project guide.

Reply:
FYI the project guid is just a Visual Studio thing. But it should be able to be left unchanged.
The one in the package manifest is what actually determines the store listing. So changing that is more disruptive.